specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -0,0 +1,5736 @@
+        "sinks",
+        "content"
+      ],
+      "type": "object",
+      "properties": {
+        "content": {
+          "description": "The script content",
+          "type": "string"
+        },
+        "language": {
+          "description": "The script language",
+          "enum": [
+            "Python"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "language",
+            "modelAsString": true
+          }
+        },
+        "revision": {
+          "format": "int32",
+          "description": "Latest script revision",
+          "type": "integer",
+          "readOnly": true
+        },
+        "scriptId": {
+          "description": "Unique id for identifying a script resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "sinks": {
+          "description": "The list of parameters the script can use as it's data sinks",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "sources": {
+          "description": "The list of parameters the script can use as it's data sources",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "visibility": {
+          "description": "Script visibility mode",
+          "enum": [
+            "ShowAll",
+            "HiddenContent"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "visibility",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Workspace": {
+      "description": "A workspace data transfer object.",
+      "required": [
+        "identity"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DefaultDto"
+        }
+      ],
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Identity Info on the Workspace"
+        },
+        "properties": {
+          "$ref": "#/definitions/WorkspaceProperties",
+          "description": "Properties on the workspace",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "WorkspaceList": {
+      "description": "List response for get Workspaces.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The Url of next result page.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Collection of items of type DataTransferObjects.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workspace"
+          }
+        }
+      }
+    },
+    "WorkspaceProperties": {
+      "description": "Workspace property bag.",
+      "type": "object",
+      "properties": {
+        "createdAt": {
```

Comment:
if you need createdAt/By lastModifiedAt/By information please implement the systemData contract.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
Will have a look, thanks!